lib/report_email_sender.rb

```diff
@@ -0,0 +1,41 @@
+class ReportEmailSender
```

Comment:
Heroku recommends switching out in their documentation hence the switch. I have no strong feelings either way.